benchmarks/styled-components/README.md

```diff
@@ -0,0 +1,21 @@
+# styled-components Benchmark
+
+Use this to benchmark `@xstyled/styled-components` against `styled-components`.
+
+## Web
```

Comment:
`@xstyled/styled-components` VS `styled-components`

Reply:
The correct english grammar is "to benchmark [something] against [something]". Since "benchmark" is being used as a verb, the proper word would be "against". To use VS it would be better to just leave

`@xstyled/styled-components` VS `styled-components`

But, as you said on the other comment, the benchmarks are not all related to `styled-components`, right? So it's better if I just remove the mention to `styled-components` entirely and use a vague benchmark description, like "Use this to benchmark `@xstyled/styled-components` against other libraries"